models/openmc/beavrs/builder.py

```diff
@@ -143,6 +143,7 @@ def get_openmc_settings_file(self):
         output = {'tallies': self.settings_output_tallies,
                   'summary': self.settings_summary}
         settings_file.output = output
+        settings_file.temperature = {'method': 'interpolation'}
```

Comment:
```suggestion
        settings_file.temperature = {'method': 'interpolation',
                                                           'default': c.operating_temperature,
                                                           'range': (300.0, 600.0),
                                                           'tolerance': 100.0}
```

It's good practice to also declare a default temperature, temperature range, and tolerance. The default ensures that all materials get a temperature (if one isn't specified). The range and tolerance minimizes the amount of data that OpenMC needs to load when initializing.